space-age/space_age.go

```diff
@@ -1,3 +1,39 @@
 package space
 
 type Planet string
+
+var (
+	earthSeconds float64 = 31557600
+)
```

Comment:
This isn't something that needs to change, so you can make it a constant:

```suggestion
const earthSeconds float64 = 31557600
```